### ISSUE TITLE
Python: Remove default system prompt in Python

### ISSUE THREAD

**Issue by DrEight** [microsoft/semantic-kernel#4200]
I don't know if it is a bug or if I'm missing something. 
I'm using the OpenAI Azure service with chatgpt 3.5. 

I was getting wrong answer from the model using SK, so I tried with the same prompt in the Azure playground using the same model, and surprise, the answer was correct. 
I'm using the same model, the same parameters like, temperature, TopP, etc, and I have an empty system message. 
I checked and double checked, and the prompt is exactly the same, but I consistently get different answers with the same prompt. 

What am I missing ?

**Comment by matthewbolanos** [comment]
Are you able to share the code that you're using? That definitely sounds strange.

**Comment by DrEight** [comment]
I think I figured out. The problem is that I wasn't aware that there is a default system prompt. 
The default system prompt is: '_Assistant is a large language model._'
I found it stepping into the Invoke call. I'm surprised that there are not examples that are mentioning the system message. It is pretty important!

Overriding the system prompt I get the expected results.

I'm using also the plugin text files config.json/skprompt.txt, but it was totally unclear to me that the message in skprompt.txt is a user message. In the examples I saw with skprompt.txt it is not mentioned the system message.
In my test with the playground I've seen that the system and the user message are treated differently.

I would suggest to add an example and documenting this behavior of SK.
